commands/core/cli.drush.inc

```diff
@@ -17,18 +22,47 @@ function cli_drush_command() {
  * Command callback.
  */
 function drush_cli_core_cli() {
-  $shell = new Psy\Shell();
+  $shell = new Shell();
 
   if (drush_drupal_major_version() >= 8) {
     // Register the assertion handler so exceptions are thrown instead of errors
     // being triggered. This plays nicer with PsySH.
-    \Drupal\Component\Assertion\Handle::register();
+    Handle::register();
     $shell->setScopeVariables(['container' => \Drupal::getContainer()]);
   }
 
+  // Add drush commands in the shell.
+  $commands = [new DrushHelpCommand()];
+
+  foreach (_drush_core_cli_get_commands() as $name => $config) {
+    $commands[] = new DrushCommand($config);
+  }
+
+  $shell->addCommands($commands);
+
   // PsySH will never return control to us, but our shutdown handler will still
   // run after the user presses ^D.  Mark this command as completed to avoid a
   // spurious error message.
   drush_set_context('DRUSH_EXECUTION_COMPLETED', TRUE);
   $shell->run();
 }
+
+/**
+ * Returns a filtered list of drush commands used for CLI commands.
```

Comment:
Please use capitalized Drush unless referring specifically to the finder script.